helpers/inspector_files.rb

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2013-2015 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of version 3 of the GNU General Public License as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE about this file by physical or electronic mail,
+# you may find current contact information at www.suse.com
+require "fileutils"
+require "cheetah"
+
+class WriteInspectorFiles
+  @@machinery = File.expand_path("../bin/machinery", File.dirname(__FILE__))
+  @@path = File.expand_path("../spec/data/", File.dirname(__FILE__))
```

Comment:
`path` is a very generic name. It would be better to reflect in the name of the variable what kind of path this is.